### PR TITLE
K8s Refresh with CRD version update

### PR DESCRIPTION
**Pull request by ggjjj**
***Please replace this line with your PR description and read PR checklist below***

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [ ] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [ ] Title of the pull request is clear and informative.
- [ ] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [ ] concise summary of tests added/modified
	- [ ] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

File: kubernetes/charts/edge-kubernetes/values.yaml [hunk 1]
```diff
@@ -221,10 +219,6 @@ edgeAgent:
     # Enables RQT data for IoT Edge Agent (disabled until metadata is finalized)
     sendRuntimeQualityTelemetry: false
 
-    # Sets timeout for operators running in Agent. Currently set to 3 minutes to guarantee correct
```
Comment (darobs) [inline kubernetes/charts/edge-kubernetes/values.yaml:224]: Add this back in

File: kubernetes/charts/edge-kubernetes/templates/iotedged-deployment.yaml [hunk 2]
```diff
@@ -39,12 +41,6 @@ spec:
           - name: "no_proxy"
             value: {{ .Values.iotedged.data.noProxy | quote}}
           {{- end}}
-          - name: "IOTEDGE_LOG"
```
Comment (darobs) [inline kubernetes/charts/edge-kubernetes/templates/iotedged-deployment.yaml:42]: Add this back in

File: kubernetes/doc/edge-deployment-to-k8s-translations.md [hunk 3]
```diff
@@ -178,7 +177,6 @@ Each IoT Edge Module will create one Deployment. This will run the module's spec
 - **nodeSelector** = `settings.k8s-extensions.nodeSelector` Placed in spec as provided.
 - **hostIPC** = `settings.createOptions.HostConfig.IpcMode` if IpcMode=host, we will set hostIPC to true
 - **hostNetwork** = `settings.createOptions.HostConfig.NetworkMode` if NetworkMode=host, we will set hostNetwork to true
-- **hostPID** = `settings.createOptions.HostConfig.PidMode` if PidMode=host, we will set hostPID to true
```
Comment (darobs) [inline kubernetes/doc/edge-deployment-to-k8s-translations.md:181]: Add line 52 & 181

File: kubernetes/charts/edge-kubernetes/values.yaml [hunk 4]
```diff
@@ -19,8 +19,6 @@ iotedged:
   data:
     enableGetNodesRBAC: true
     targetPath: /var/lib/iotedge
-    # set iotedged log level.
-    runtimeLogLevel: "info"
```
Comment (darobs) [inline kubernetes/charts/edge-kubernetes/values.yaml:23]: Add this back in.

File: kubernetes/charts/edge-kubernetes-crd/Chart.yaml [hunk 5]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for installing CRD for Azure IoT Edge on Kubernetes
 name: edge-kubernetes-crd
-version: 0.2.13
+version: 0.2.11
```
Comment (darobs) [inline kubernetes/charts/edge-kubernetes-crd/Chart.yaml:5]: Should be 0.2.14

File: kubernetes/charts/edge-kubernetes/Chart.yaml [hunk 6]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for running Azure IoT Edge on Kubernetes
 name: edge-kubernetes
-version: 0.2.13
```
Comment (darobs) [inline kubernetes/charts/edge-kubernetes/Chart.yaml:5]: Shoudl be 0.2.14

File: builds/misc/images.yaml [hunk 7]
```diff
@@ -2,439 +2,9 @@ trigger:
...
-          project: CloudToDeviceMessageTester
-
-     # Metrics Collector (customer-facing)
-      - template: templates/image-linux.yaml
-        parameters:
-          name: Metrics Collector (customer-facing)
-          imageName: azureiotedge-metrics-collector
-          project: Microsoft.Azure.Devices.Edge.Azure.Monitor
-
-################################################################################
```
Comment (darobs) [inline builds/misc/images.yaml:189]: We'll need to bring back deleted lines 189-211 here. Jobs with "templates/image-linux-rust-jobs.yaml"

File: kubernetes/charts/edge-kubernetes/values.yaml [hunk 8]
```diff
@@ -206,7 +208,7 @@ edgeAgent:
     # storageClassName: ""
 
     # Set this to true if you wish to turn on experimental features support. Please see 
-    # https://github.com/Azure/iotedge/blob/release/1.1/kubernetes/doc/create-options.md to
+    # https://github.com/Azure/iotedge/blob/master/kubernetes/doc/create-options.md to
```
Comment (ggjjj) [inline kubernetes/charts/edge-kubernetes/values.yaml:211]: whats the right link @darobs

File: builds/misc/templates/build-edgelet-linux.yaml [hunk 9]
```diff
@@ -1,15 +0,0 @@
-parameters:
```
Comment (darobs) [inline builds/misc/templates/build-edgelet-linux.yaml:1]: Bring this back

**Comment by ggjjj** [comment]
Closing this as we are not planning on releasing images with security updates

**Pull request closed without merging**